website/source/docs/platform/k8s/ambassador.html.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: "docs"
+page_title: "Ambassador Integration - Kubernetes"
+sidebar_current: "docs-platform-k8s-ambassador"
+description: |-
+    Ambassador is a Kubernetes-native API gateway and ingress controller that 
+    integrates well with Consul Connect.
+---
+
+# Ambassador Integration with Consul Connect
+
+Besides enabling Kubernetes services to discover and securely connect to each other,
+Connect also can help route traffic into the cluster from outside, when paired with 
+an [ingress controller] like DataWire's Ambassador.
+ 
+[Ambassador] is a popular Kubernetes-native service that acts as an ingress controller
+or API gateway. It supports an optional integration with Consul that allows it to
+route incoming traffic to the [proxies] for your Connect-enabled services. 
+
+This means you can have **end-to-end encryption** from the browser, to Ambassador,
+to your Kubernetes services. 
+
+
+## Installation 
+
+First, install Consul and enable Connect on the agents inside the cluster. Decide
+whether you will enable [service sync] or manually register your services with
+Consul.
+
+Once you have tested and verified that everything is working, you can proceed with
+the Ambassador installation. Full instructions are available on the [Ambassador
+site][install], but a summary of the steps is as follows:
+
+If you are deploying to GKE, create a `RoleBinding` to grant you cluster admin
```

Comment:
```suggestion
If you are deploying to GKE, create a `RoleBinding` to grant your cluster admin
```

Reply:
"Cluster" is also a pretty overloaded term for us, so you might say which cluster is getting admin rights to what.

Reply:
I think it's clear to Kubernetes users in this context, since `RoleBinding` is a k8s concept and not a Consul concept. Also, it is actually "grant you rights" and not "grant your rights". The rights are being granted to the logged-in user, i.e. you.